packages/mail/lib/sendMailTemplate.js

```diff
@@ -210,7 +213,7 @@ module.exports = async (mail, context, log) => {
     subject: mail.subject,
     from_email: mail.fromEmail || DEFAULT_MAIL_FROM_ADDRESS,
     from_name: mail.fromName || DEFAULT_MAIL_FROM_NAME,
-    html: getTemplate(mail.templateName),
+    ...getTemplates(mail.templateName),
     merge_language: mail.mergeLanguage || 'handlebars',
     global_merge_vars: mergeVars,
     auto_text: true,
```

Comment:
set `auto_text` to false if the text template is available

Reply:
Done, too.